frontend/src/Components/Login/Login.css

```diff
@@ -0,0 +1,15 @@
+.imageLogo {
```

Comment:
I thinnk I saw classes with `-` and now here with camelCase, just choose one style through project.

Reply:
We wanted to separate Bootstrap styles with our custom style to know at any point, by just looking at it, if that's our custom style or bootstrap one.
But from now on we'll be consistent with naming them.

Reply:
You can come up with a prefix on your custom classes like f.e. SkyNest -> sn-_class_name_. We do sth similar on projetc also.